.setup/bin/setup_sample_courses.py

```diff
@@ -1441,6 +1441,7 @@ def __init__(self, gradeable):
             self.submission_open_date = dateutils.parse_datetime(gradeable['eg_submission_open_date'])
             self.submission_due_date = dateutils.parse_datetime(gradeable['eg_submission_due_date'])
             self.team_lock_date = dateutils.parse_datetime(gradeable['eg_submission_due_date'])
+            self.regrade_request_start_date = dateutils.parse_datetime(gradeable['eg_regrade_request_start_date'])
             self.regrade_request_date = dateutils.parse_datetime(gradeable['eg_regrade_request_date'])
```

Comment:
We have changed the term from "regrade request" to "grade inquiry" on the user interface.  We left the legacy name of regrade request in the code, but I think now is the time to replace that.

So please change "regrade request" to "grade_inquiry" everywhere in the code.

And since we are adding a second date for regrade request/grade inquiry, let's rename the original date from "regrade_request_date" to "grade_inquiry_due_date" to avoid future confusion.  Do this everywhere.